no-computer.md

```diff
@@ -11,34 +12,64 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+While I was thinking about my first free-response question, I realized that
+(written) chemical reactions are already a DSL.  Thus the group served by them
+is chemists, and their experience is that they can look at chemical reactions
+in isolation and more academically than in a lab setting.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+This DSL is necessary because it allows for a clear, unambiguous way of 
+describing a reaction and how it should occur without having to actually
+observe or test the reaction.  It meets that need by showing exactly what is
+or should be happening in the reaction.
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I really like this because I love chemistry, and I love programming.  It came to
+me when I was thinking of my first DSL, which seems like it would be a superset
+of typical chemical reactions.
+
 
 ### Domain
 _Describe the project's domain in five words._
 
+Chemical reactions and their products
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+Written on paper/typed in some way/drawn out.  Programming looks like depicting
+the molecules on some 2D surface and describing what they should result in.  
+This is the right way because it simplifies and abstracts away a lot of the 
+important, but not necessarily directly related stuff associated with chemistry.
+
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+You don't really run the reaction (on paper), although when you run it in a lab
```

Comment:
I would say that since you can't run the reaction (on paper), this is more of a communication-oriented language than a programming language. Though it's certainly still a DSL!